cli/tauri.js/src/helpers/spawn.ts

```diff
@@ -1,24 +1,26 @@
-const logger = require('./logger')
+import crossSpawn from 'cross-spawn'
+import logger from './logger'
+
 const log = logger('app:spawn')
 const warn = logger('app:spawn', 'red')
-const crossSpawn = require('cross-spawn')
 
 /*
- Returns pid, takes onClose
+  Returns pid, takes onClose
  */
-module.exports.spawn = function (cmd, params, cwd, onClose) {
+export const spawn = (
+  cmd: string,
+  params: string[],
+  cwd: string,
+  onClose: (code: number) => void
+): number => {
   log(`Running "${cmd} ${params.join(' ')}"`)
   log()
 
-  const runner = crossSpawn(
-    cmd,
-    params, {
-      stdio: 'inherit',
-      stdout: 'inherit',
-      stderr: 'inherit',
```

Comment:
Did you remove those options intentionally?

Reply:
According to the crossSpawn typings, those were invalid options. I should have ignored them and then added a comment. However, stdio covers stdin/out/error, so it should have the same behavior.